### PR TITLE
doc(*): update api docs and added missing the API of groups

### PR DESCRIPTION
**Pull request by raoxiaoyan**
### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

FTI-5986

### CODE REVIEW

**Comment by lena-larionova** [comment]
@raoxiaoyan is this for 3.7?

**Comment by raoxiaoyan** [comment]
> @raoxiaoyan is this for 3.7?
This PR will affect all versions. Some examples were wrong. I need to correct it.
@lena-larionova How can I get the updated docs? The `https://deploy-preview-7436--kongdocs.netlify.app/` still is the old docs. I want to make sure I updated it correctly.

**Comment by Guaris** [comment]
> > @raoxiaoyan is this for 3.7?
> > This PR will affect all versions. Some examples were wrong. I need to correct it.
> > @lena-larionova How can I get the updated docs? The `https://deploy-preview-7436--kongdocs.netlify.app/` still is the old docs. I want to make sure I updated it correctly.


There are two ways to preview your changes: 
1. Copy the edited spec and paste it here https://editor.swagger.io/

2. Create a Konnect organization and load your edited spec and see the rendered result.

**Comment by raoxiaoyan** [comment]
> > > @raoxiaoyan is this for 3.7?
> > > This PR will affect all versions. Some examples were wrong. I need to correct it.
> > > @lena-larionova How can I get the updated docs? The `https://deploy-preview-7436--kongdocs.netlify.app/` still is the old docs. I want to make sure I updated it correctly.
> 
> There are two ways to preview your changes:
> 
> 1. Copy the edited spec and paste it here https://editor.swagger.io/
> 2. Create a Konnect organization and load your edited spec and see the rendered result.

Thank you.

File: api-specs/Gateway-EE/latest/kong-ee.yaml [hunk 1]
```diff
@@ -14762,6 +15132,9 @@ tags:
   - description: |
       Admin routes
     name: admins
+  - description: |
+      Group routes
+    name: groups
   - description: |
       Debug Routes
     name: debug
```
Comment (sumimakito) [inline api-specs/Gateway-EE/latest/kong-ee.yaml:15140]:
```suggestion
  - description: |
      Admin routes
    name: Admins
  - description: |
      Group routes
    name: Groups
  - description: |
      Debug Routes
    name: Debug
```

Shall we capitalize tag names to keep consistent with other tags?

Screenshot 2024-06-03 at 16 19 27

**Review by sumimakito — APPROVED** [review]
Verified with Swagger Editor and LGTM. Left a non-blocking comment.

File: api-specs/Gateway-EE/latest/kong-ee.yaml [hunk 2]
```diff
@@ -12586,32 +12586,35 @@ paths:
...
-                          name:
-                            type: string
-                      workspace:
-                        type: object
-                        properties:
-                          id:
-                            type: string
-                  next:
-                    nullable: true
+                    items:
```
Comment (raoxiaoyan) [inline api-specs/Gateway-EE/latest/kong-ee.yaml:12589]: Why is there `items` here? 
image

**Comment by raoxiaoyan** [comment]
> @raoxiaoyan I made an edit to the delete endpoint, initially it had a request body, but I believe you meant to make those parameters, is that right?

Nice catch.

**Review by Guaris — APPROVED** [review]

**Pull request merged**